rwengine/src/engine/GameWorld.cpp

```diff
@@ -678,6 +678,26 @@ int GameWorld::getMinute()
 	return state->basic.gameMinute;
 }
 
+void GameWorld::offsetGameTime(int minutes)
+{
+	int gameMinute = state->basic.gameMinute;
+	int gameHour   = state->basic.gameHour;
+
+	gameMinute += minutes;
+	bool backwards = gameMinute < 0;
+
+	gameHour   += gameMinute / 60 - (backwards ? 1 : 0);
```

Comment:
As this is the only use of `backwards` I feel this could be more expressive:

```
// If the target minute is positive we only add 1 hour per 60 minutes as it represents a wrap of the minutes.
gameHour += gameMinute / 60;

// However, if the target minute is negative, there was an additional wrap already (from 00 to 59)
if (gameMinute < 0) {
  gameHour -= 1;
}
```

This isn't perfect though. I don't feel like bikeshedding over this, but maybe something to consider.
(I personally felt the backwards part was hard to understand in your code)